src/simsopt/util/fourier_interpolation.py

```diff
@@ -52,3 +53,55 @@ def fourier_interpolation(fk, x):
 
     # Evaluate interpolant as matrix-vector products
     return np.dot(D, w * fk) / np.dot(D, w)
+
+
+def sin_coeff(f, n, L=pi, maxiter=2000, tol=1.48e-8):
```

Comment:
Why can't we use the scipy.fft or fftw3 library here? What is the need for separate functions?

Reply:
Well it's not that we can't, but the way I have done it is to the best of my knowledge the most straight forward way.  scipy.fft is implemented in the complex exponential form and also adds the intricacy of manipulating the data before performing the fft so that the right fundamental frequency is picked. Nonetheless, I will try to implement it with the fft and get back to you.